From 480d5745e1443de6e91a9f9c42d94e29705a3f43 Mon Sep 17 00:00:00 2001
From: ysau <yatshan.au@gmail.com>
Date: Fri, 21 Jul 2017 16:29:12 -0700
Subject: [PATCH] # Issue 17046 # #    modified:
 /pandas/tests/indexes/test_base.py, #             
 /doc/source/whatsnew/v0.21.0.txt # # Include both '>' and '<' in the error
 message in # TestMixedIntIndex.test_argsort and #
 TestMixedIntIndex.test_numpy_argsort

Modified: Mo Zhou <cdluminate@gmail.com>
Modification Note: Part of the original patch doesn't apply.
 Removed modification to doc/source/whatsnew/v0.21.0.txt from the original
 patch.

diff --git a/pandas/tests/indexes/test_base.py b/pandas/tests/indexes/test_base.py
index 692cdd49579..842e8fea0df 100644
--- a/pandas/tests/indexes/test_base.py
+++ b/pandas/tests/indexes/test_base.py
@@ -1846,7 +1846,7 @@ def create_index(self):
     def test_argsort(self):
         idx = self.create_index()
         if PY36:
-            with tm.assert_raises_regex(TypeError, "'>' not supported"):
+            with tm.assert_raises_regex(TypeError, "'>|<' not supported"):
                 result = idx.argsort()
         elif PY3:
             with tm.assert_raises_regex(TypeError, "unorderable types"):
@@ -1859,7 +1859,7 @@ def test_argsort(self):
     def test_numpy_argsort(self):
         idx = self.create_index()
         if PY36:
-            with tm.assert_raises_regex(TypeError, "'>' not supported"):
+            with tm.assert_raises_regex(TypeError, "'>|<' not supported"):
                 result = np.argsort(idx)
         elif PY3:
             with tm.assert_raises_regex(TypeError, "unorderable types"):
